### ISSUE TITLE
rig ba

### ISSUE THREAD

**Issue by zhouzhimin-hz** [colmap/colmap#1185]
Hi:
  in bundle_adjustment.cc line 1063, problem_->SetParameterization(tvec_data, tvec_parameterization);
I think it should be problem_->SetParameterization(rig_tvec_data, tvec_parameterization); is it right?

**Comment by sniradi-limar** [comment]
No, qvec_data and tvec_data are only used for images not part of a rig. See https://github.com/colmap/colmap/blame/96d4ba0b55c0d1f98c8c432420ecd6540868c398/src/optim/bundle_adjustment.cc#L956

**Comment by shreyas23** [comment]
No, qvec_data and tvec_data are only used for images not part of a rig. See https://github.com/colmap/colmap/blame/96d4ba0b55c0d1f98c8c432420ecd6540868c398/src/optim/bundle_adjustment.cc#L956
